CMakeLists.txt

```diff
@@ -6,8 +6,8 @@ project(viam-mlmodelservice-triton
   LANGUAGES CXX
 )
 
-# Use of the C++ SDK requires C++14.
-set(CMAKE_CXX_STANDARD 14)
+# Use of the C++ SDK requires C++14 or later. We make use of std::filesystem, which requires C++17 or later.
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Does the jetpack5 platform support this?

Reply:
Yes, building with both `docker build . -f etc/docker/Dockerfile.triton-jetpack-focal` and `docker build . -f etc/docker/Dockerfile.nvcr-triton-containers` succeeds. C++ 17 was released long before Jetpack5 came out, so I don't think this is surprising.